src/test/java/com/github/fishio/TestEnemyFish.java

```diff
@@ -83,10 +86,10 @@ public void testonCollide() {
 	 */
 	@Test
 	public void testlimitVx1() {
-		Vec2d vec1 = new Vec2d(EnemyFishFactory.MAX_EFISH_SPEED + 1, 2);
```

Comment:
*Auto-generated comment*

    
    The String literal "MAX_EFISH_SPEED" appears 16 times in this file; the first occurrence is on line 89